pkg/kv/kvserver/helpers_test.go

```diff
@@ -495,15 +495,7 @@ func (r *Replica) GetQueueLastProcessed(ctx context.Context, queue string) (hlc.
 }
 
 func (r *Replica) MaybeUnquiesce() bool {
```

Comment:
The semantics of `MaybeUnquiesce` changed. Were there any callers who wanted the old behaviour (wakeLeader=mayCampaign=false)?